.github/workflows/build-mudlet.yml

```
@@ -194,6 +201,17 @@ jobs:
          echo "WITH_3DMAPPER=no" >> $GITHUB_ENV
          echo "WITH_FONTS=no" >> $GITHUB_ENV
 
+    - name: Set Sentry environment variables
+      run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:27: Double quote to prevent globbing and word splitting

(shellcheck)